index.js

```diff
@@ -1321,13 +1323,13 @@ Auth0Lock.prototype._signinPopupNoRedirect = function (connectionName, popupCall
       self._showError(self.options.i18n.t('signin:userConsentFailed'));
     } else if (err.status === 0) {
       self._showError(self.options.i18n.t('networkError'));
-    } else if (err.status !== 401) {
-      self._showError(self.options.i18n.t('signin:serverErrorText'));
-    } else if ('unauthorized' === err.code && err.details && err.details.error_description === 'user is blocked') {
+    } else if ('too_many_attempts' === err.code) {
       message = self.options.i18n.t('signin:userBlockedErrorText');
-      self._showError(message || err.details.error_description);
+      self._showError(message || err.message);
```

Comment:
A different key should be used for this error, for reset we use `tooManyRequestsErrorText`.
Also, we should always get the text from the dict (we only allow to display the message returned by the server in some special cases involving rules).